data/fields.py

```diff
@@ -72,12 +72,12 @@ class Base64BinaryField(TextField):
     def db_value(self, value):
         if value is None:
             return None
-        return base64.b64encode(value)
+        return base64.b64encode(value).decode('ascii')
```

Comment:
I ran into this in my conversion work as well. This means every time we want to use this field, we need to make sure we are expecting actual `bytes` when using this field. Does that make sense? Or should we just keep everything as strings?

To avoid headaches, this guy suggested using the ["Unicode Sandwich"](https://nedbatchelder.com/text/unipain.html#h_pain_relief) (decode incoming binary immediately -> Python code manipulates decoded strings -> encode outgoing strings it

Reply:
One potential pitfall is that not all unicode can be accurately converted to ASCII characters. We should verify the behavior on a case-by-case basis. My hunch is that it's fine but it'd be worth verifying the data being base64 encoded.

Reply:
If the data is passed through `b64encode`, it should be fine, as that call will fail if it cannot encode the data